packages/test-utils/src/mocks/asset/v1/data.ts

```diff
@@ -0,0 +1,71 @@
+import { addYears, parseISO } from "date-fns";
+import faker from "faker";
+import { Asset, AssetTenure } from "@mtfh/common/lib/api/asset/v1";
+import { TENURE_TYPES } from "../../../constants";
+
+export const generateMockAssetTenureV1 = (
+  data: Partial<AssetTenure> = {},
+): AssetTenure => {
+  const type =
+    data.type || faker.random.arrayElement(TENURE_TYPES.map(({ value }) => value));
+  const isActive = data.isActive !== undefined ? data.isActive : faker.datatype.boolean();
+  const startOfTenureDate = data.startOfTenureDate
+    ? parseISO(data.startOfTenureDate)
+    : faker.date.between("2010-01-01", "2020-01-01");
+  return {
+    id: faker.datatype.uuid(),
+    paymentReference: faker.random.alphaNumeric(10),
+    startOfTenureDate: startOfTenureDate.toISOString(),
+    endOfTenureDate: !isActive
+      ? addYears(startOfTenureDate, 1).toISOString()
+      : addYears(startOfTenureDate, 100).toISOString(),
+    type,
+    isActive,
+  };
+};
+
+export const generateMockAssetV1 = (data: Partial<Asset> = {}): Asset => ({
+  id: faker.datatype.uuid(),
+  assetId: faker.datatype.uuid(),
+  assetType:
+    data.assetType || faker.random.arrayElement(["Dwelling", "LettableNonDwelling"]),
```

Comment:
Spreading data gives us more flexibility to use the generator
```ts
({
   assetType:  faker.random.arrayElement(["Dwelling", "LettableNonDwelling"]),
  ...data,
})
```

Reply:
good point yep